osmaxx-py/excerptconverter/dummyexcerptconverter/dummy_excerpt_converter.py

```diff
@@ -73,10 +75,10 @@ def create_output_files(execution_configuration, extraction_order, supported_exp
 
             if private_storage.exists(file_name):
                 message_text = _('"%s" created successful' % file_name)
-                BaseExcerptConverter.inform_user(extraction_order.orderer, messages.SUCCESS, message_text, False)
+                converter_helper.inform_user(messages.SUCCESS, message_text, False)
             else:
                 message_text = _('Creation of "%s" failed!' % file_name)
-                BaseExcerptConverter.inform_user(extraction_order.orderer, messages.ERROR, message_text, False)
+                converter_helper.inform_user(messages.ERROR, message_text, False)
```

Comment:
Always use named arguments for boolean parameters. See http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html